### PR TITLE
add scraper for European Centre for Disease Prevention and Control

### PR DESCRIPTION
**Pull request by Runinho**
Sadly this website is quite ugly to parse. 
I have not spotted an error but i would appreciate if someone can check.

### CODE REVIEW

**Review by Timoeller — CHANGES_REQUESTED** [review]
Hey thanks, yes, this looks like the most complex crawler we have so far : ) Good work!

I have made some comments where we need to adjust to latest changes. Could you change them, I will merge it then tomorrow morning.

File: data/scrapers/ECDC_scraper.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    columns["category"] = [""] * len(columns["question"])
+    columns["country"] = [""] * len(columns["question"])
+    columns["region"] = [""] * len(columns["question"])
+    columns["city"] = [""] * len(columns["question"])
+    columns["lang"] = ["en"] * len(columns["question"])
+    columns["last_update"] = [today.strftime("%Y/%m/%d")] * len(columns["question"])
+
+    dataframe = pd.DataFrame(columns)
+
+    dataframe.to_csv("ecdc_en.tsv", sep="\t", index=False)
```
Comment (Timoeller) [inline data/scrapers/ECDC_scraper.py]: please do not write to disk but return the columns, we have a meta scraper now

File: data/scrapers/ECDC_scraper.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      columns["answer"].append(answer)
+      columns["answer_html"].append(answer_html)
+
+    today = date.today()
+
+    columns["link"] = ["https://www.ecdc.europa.eu/en/novel-coronavirus-china/questions-answers"] * len(columns["question"])
+    columns["name"] = ["Q & A on COVID-19"] * len(columns["question"])
+    columns["source"] = ["European Centre for Disease Prevention and Control"] * len(columns["question"])
+    columns["category"] = [""] * len(columns["question"])
+    columns["country"] = [""] * len(columns["question"])
```
Comment (Timoeller) [inline data/scrapers/ECDC_scraper.py:63]: add EU or something else. We will filter by these vals?

File: data/scrapers/ECDC_scraper.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# run 'scrapy runspider ECDC_scraper.py' to scrape data
+
+from datetime import date
+import scrapy
+import pandas as pd
+from scrapy.crawler import CrawlerProcess
+
+class ECDCScraper(scrapy.Spider):
```
Comment (Timoeller) [inline data/scrapers/ECDC_scraper.py:8]: please name this class "CovidScraper" so the metacrawler can access it.

File: data/scrapers/ECDC_scraper.py (hunk 1 above)
Comment (Runinho) [reply data/scrapers/ECDC_scraper.py]: ah, nice! changed it

**Review by Timoeller — APPROVED** [review]
nice thanks. You missed to change the actual class name, but I fixed and merged it.

**Pull request merged**